src/utils/map-state.js

```diff
@@ -2,16 +2,16 @@ import {PerspectiveMercatorViewport} from 'viewport-mercator-project';
 import assert from 'assert';
 
 // MAPBOX LIMITS
-const MAX_PITCH = 60;
-const MAX_ZOOM = 20;
+export const MAPBOX_MAX_PITCH = 60;
+export const MAPBOX_MAX_ZOOM = 20;
```

Comment:
Q: Is this still a MAPBOX default limit? I.e. is it reasonable to call it `MAPBOX_`?

Putting `MAPBOX` constants is ok, although one could argue that in many ways this class implements completely completely generic (while it implements a compatible interaction model, the critical mapbox synchronization is handled by the Viewport).